internal/luasandbox/sandbox.go

```diff
@@ -32,7 +35,42 @@ func (s *Sandbox) RunScript(ctx context.Context, opts RunOptions, script string)
 	ctx, endObservation := s.operations.runScript.With(ctx, &err, observation.Args{})
 	defer endObservation(1, observation.Args{})
 
+	return s.RunScriptNamed(ctx, opts, singleScriptFS{script}, "main.lua")
+}
+
+type FS interface {
+	ReadFile(name string) ([]byte, error)
```

Comment:
Seems like that's the only use right now.

```suggestion
	ReadFile(name string) (string, error)
```